steelthread-0/walkthrough.md

```diff
@@ -15,123 +18,295 @@ The installation has already been performed on an OpenShift cluster as per the i
+docker push docker.io/<dockerhubusername>/che-node
 ```
 
-# Setup HTTP connection to the messaging booster in Fuse Ignite
-- Open the Fuse ignite console and open the connections again. 
-- Click create connection
-- Find the HTTP connection
-- Set the base URL as the url exposed from the messaging front end application
-- Click validate to ensure Fuse Ignite can reach the endpoint
-- Click next 
-- Give the connection a name (booster)
-- Click create
+**IMPORTANT:** You will need to make the `che-node` image repository **public** in your Docker Hub account. You can do this from the Docker Hub web console.
 
-# Creating the integration
+![Docker Hub Public Repository](./docker_hub_public_repository.png)
 
-- Open the Fuse Ignite console
-- Click on integrations
-- Click create integration
-- Click on the Enmaase connection
-- Click on the ``` Subscribe for messages ``` option
-- Under Destination Name add ``` work-queue/worker-updates ```
-- Choose Queue as the Destination Type
-- Click Next
-- Set the Specify Output Data Type as ``` Type specification not required ``` this is the default by clicking done
-- On the ``` Choose a Finish Connection ``` screen select the messaging booster connection you created
-- Choose ```Invoke URL ``` 
-- In the URL Path set the path to ```/api/data```
-- In the method select ```GET``` and click next
-- Again under Specify Input Data Type leave it as ``` Type specification not required ``` and click done
-- Click publish at the top right
-- Choose the integration name (steel thread)
+Back in the Che Dashboard, go to the Stacks menu & clone the Node stack
+
+![Che Clone Stack](./che_stack_clone.png)
+
+Modify the stack to set the Machine image to the public image repository name i.e. `<dockerhubusername>/che-node`
+
+![Che Machine Image](./che_stack_machine_image.png)
+
+This stack can then be used to create a Workspace later.
+
+### Import the self signed certificate into your browser
+
+Before you can use a Workspace from your browser, the self-signed certificate needs to be imported. This allows the secure websocket connection to be established when you are in the Che IDE.
+To do this in Chrome, open `chrome://settings/certificates`, then `Authorities > Import`. Choose the ca.crt file and import it.
+You can check if it was imported correctly and is the correct cert when using a Workspace later. There should be no red errors related to `wss://` connections.
+
+## Customise the messaging booster
+
+With the Che setup done, we can create a workspace and modify the booster code.
+The modification will add information to messages as they get sent from the messaging booster. To do this we will need to change some of the code in the messaging booster. 
+
+- Login to the Che Dashboard. The url is in the format ```https://che-che.<ROUTER_SUFFIX>``` 
+- Create a new workspace from the custom Node.js Stack (the stack that has the custom image with self-signed cert included)
```

Comment:
Think we need more detail here.

- Create a new workspace and select the custom Node.js Stack from the list of stacks (the stack that has the custom image with self-signed cert included)
- Click Add or Import Project button
- Click the Github tab
- Select the messaging-frontend booster repo that was created by Launcher and click the Add button
- Click Create button at the bottom of the screen
- When prompted, click the Open In IDE button